### PR TITLE
Create new CSS file

### PR DESCRIPTION
**Pull request by franco999**
CSS template for HTML5 code

### CODE REVIEW

**Comment by frlan** [comment]
I think it's more suitable for the wiki instead of deploying it everywhere.

**Comment by elextr** [comment]
Well, we have an HTML5 template, so it makes sense (to me) to have a matching css.

**Comment by kugel-** [comment]
Agree with @elextr "new file" templates are very suitable for Geany itself, especially as Geany does pattern substitution.

There should be a {fileheader} token at the start, to insert license and stuff.

File: data/templates/files/file.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+header{}
+nav{}
+nav a:link,nav a:visited{}
+nav a:hover{}
+section{}
+article{}
+footer{}
+
+/* Identifiers */
+#IDhere{}
```
Comment (b4n) [inline data/templates/files/file.css]: I'm not really fond of such placeholders, but maybe it doesn't matter.

**Comment by b4n** [comment]
I don't have a strong opinion here.  If it's useful enough, why not.

**Comment by franco999** [comment]
Sorry for my English. - Google translator -
Here, I just wanted to take an example, the CSS file to add it as a template. If you need to make changes to better suit the general public, you can do. It's just my humble example.

**Comment by b4n** [comment]
See #2259

**Pull request closed without merging**